### PR TITLE
testing/iozone: new aport

### PR DESCRIPTION
**Pull request by OTLabs**
http://iozone.org
IOzone filesystem benchmark

### CODE REVIEW

File: testing/iozone/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		s390x*) target="linux-S390X" ;;
+		ppc64le*) target="linux-powerpc64" ;;
+		*) msg "Unable to determine architecture from (CARCH=$CARCH)" ; return 1 ;;
+	esac
+
+	make $target CFLAGS='-DHAVE_PREADV -DHAVE_PWRITEV'
+}
+
+package() {
+	cd "$builddir"
```
Comment (tcely) [inline testing/iozone/APKBUILD]:
```suggestion
```

Comment (OTLabs) [reply testing/iozone/APKBUILD]: Done! Thank you!

**Comment by Ikke** [comment]
When the license is custom, you [need to include it in the doc subpkg][custom-license]. Because there is no separate license file, perhaps something like this could work:

```diff
diff --git a/testing/iozone/APKBUILD b/testing/iozone/APKBUILD
index fc0c846fbf..3e416de506 100644
--- a/testing/iozone/APKBUILD
+++ b/testing/iozone/APKBUILD
@@ -9,6 +9,7 @@ url="http://iozone.org"
 arch="all"
 license="custom"
 options="!check" # No test suite from upstream
+subpackages="$pkgname-doc"
 source="http://iozone.org/src/current/${pkgname}${_pkgver}.tar"
 builddir="$srcdir/${pkgname}${_pkgver}/src/current/"

@@ -31,6 +32,9 @@ build() {
 package() {
        install -m755 -D iozone \
                "$pkgdir"/usr/bin/iozone
+
+       install -dm755 "$pkgdir"/usr/share/licenses/$pkgname/
+       sed -n '1,49p' "$builddir"/iozone.c >"$pkgdir/usr/share/licenses/$pkgname/LICENSE"
 }

 sha512sums="2f3a9d259ca531e45124e8ad55358706e2f9615f50fbf434e786b832622aa5f673dbed1b2706c6450c200e0af7f7374e40402e0b06ad116d5f2a0704a6156a1a  iozone3_487.tar"
```
I'm not sure if this is acceptable though.

[custom-license]:https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license

**Comment by OTLabs** [comment]
@Ikke Thank you for solution template. I have implemented it.

**Comment by OTLabs** [comment]
@tcely Implemented `-doc` sub package, included LICENSE as @Ikke suggested.

I have also contacted the author asking about possibility to include a license in separate file for future releases.

**Comment by Ikke** [comment]
This has been merged as 318fb2e, closing PR.

**Pull request closed without merging**